funcs_scripts.php

```diff
@@ -88,14 +108,30 @@ function delete_file_if_exists($filename)
 function module_is_recipe()
 {
     global $MODULE_DIR;
-    if (strpos('/recipe-', $MODULE_DIR) !== false
-        || strpos('/silverstripe-installer', $MODULE_DIR) !== false
+    if (strpos($MODULE_DIR, '/recipe-') !== false
+        || strpos($MODULE_DIR, '/silverstripe-installer') !== false
     ) {
         return true;
     }
     return false;
 }
 
+/**
+ * Determine if the module being processed is something installed on a website e.g. gha-*
+ *
+ * Example usage:
+ * module_is_not_for_cms()
+ */
+function module_is_not_for_cms()
+{
+    global $MODULE_DIR;
+    return strpos($MODULE_DIR, '/gha-') !== false
+        || strpos($MODULE_DIR, '/developer-docs') !== false
+        || strpos($MODULE_DIR, '/vendor-plugin') !== false
+        || strpos($MODULE_DIR, '/eslint-config') !== false
+        || strpos($MODULE_DIR, '/webpack-config') !== false;
+}
```

Comment:
vendor/plugin is a weird one. I wouldn't say it's used by the cms, instead I'd say it's used by composer

It's a requirement of silverstripe/installer so that can do things like run `composer vendor-expose`